simple_history/utils.py

```diff
@@ -134,15 +141,20 @@ def bulk_update_with_history(
         in each historical record
     :param default_date: Optional date to specify as the history_date in each historical
         record
+    :param manager: Optional model manager to use for the model (default is `objects`)
     """
     if django.VERSION < (2, 2,):
         raise NotImplementedError(
             "bulk_update_with_history is only available on "
             "Django versions 2.2 and later"
         )
     history_manager = get_history_manager_for_model(model)
+    model_manager = manager or model.objects
```

Comment:
Think you can make it the default manager by default rather than objects. That's my bad for not catching that  https://docs.djangoproject.com/en/3.1/topics/db/managers/#default-managers

Reply:
Ah yeah, that's even better then. I wonder if it's then not necessary to include the optional parameter to `bulk_create_with_history` because a manager will always be available via the default? The only reason I added it there is in case `objects` doesn't exist.

`bulk_update_with_history` will still require the parameter.

Reply:
Yep, that works for me.

Reply:
Made the changes and updated the docs.